src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/matchxpath/InputMatchXpathDoubleBrace.java

```diff
@@ -0,0 +1,18 @@
+package com.puppycrawl.tools.checkstyle.checks.coding.matchxpath;
+
+import java.util.ArrayList;
+
+/* Config:
+ *
+ * query: //INSTANCE_INIT[not(../*[not(self::LCURLY or self::INSTANCE_INIT or self::RCURLY or
+ self::SINGLE_LINE_COMMENT or self::BLOCK_COMMENT_BEGIN)])]
```

Comment:
please add leading asterisk, we almost merged Checks for this - #9006.

Reply:
done